### ISSUE TITLE
Header / footer logo accessibility

### ISSUE THREAD

**Issue by justinlittman** [sul-dlss/happy-heron#3177]
The image link with the logo for Stanford Libraries has an aria-label in addition to an image with alternative text and both instances have a value of "Stanford Digital Repository" without including the visible Libraries text. (also: "There are images with an alternative text that does not provide a meaningful equivalent. The "Stanford Libraries" logo has "Stanford Digital Repository" as the alternative text.")

**Comment by justinlittman** [comment]
@astridu The logo is:
<img width="223" alt="image" src="https://github.com/sul-dlss/happy-heron/assets/588335/9ea5d944-8337-4f11-aa69-1f8d5d065f19">

However, it links to https://sdr.stanford.edu/

That seems kinda contradictory (i.e., an image for SUL Libraries that links to SDR) and makes it unclear how to correct the labels.

**Comment by astridu** [comment]
Agree. Let's have the logo link to Stanford Libraries. I looked at other sites, and it looks like they link the 'Stanford University' logo back to the main university page, and then their department name 'Stanford | Engineering' goes to the engineering home. Could we do something similar here where our logo links to Stanford Libraries, and then our name 'Stanford | Digital Repository" goes to SDR home?

**Comment by justinlittman** [comment]
That'll work @astridu.
